src/main/java/game/GameEngine.java

```diff
@@ -44,7 +47,7 @@ public static void main(String[] args) {
                     } else {
                         loadMap(filePath, map);
                     }
-                    editMap(map, filePath);
+                   editMap(map, filePath);
```

Comment:
could you revert this please, as you have changed the indentation which is now not looking good?

Reply:
completed